runtime/common/src/crowdloan/migration.rs

```diff
@@ -15,26 +15,27 @@
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
 
 use super::*;
-use frame_support::{generate_storage_alias, Twox64Concat};
+use frame_support::{storage_alias, Twox64Concat};
 
 /// Migrations for using fund index to create fund accounts instead of para ID.
 pub mod crowdloan_index_migration {
 	use super::*;
 
+	#[storage_alias]
+	type NextTrieIndex<T: Config> = StorageValue<Pallet<T>, FundIndex>;
+
+	#[storage_alias]
+	type Leases<T: Config> = StorageMap<Pallet<T>, Twox64Concat, ParaId, Vec<Option<(<T as frame_system::Config>::AccountId, BalanceOf<T>)>>>;
```

Comment:
Is the prefix for this correct? Previously it had `Slots`, but this one now has `Pallet<T>`, which I assume is the one for `Crowdloan`?

Reply:
Ohh! Could find!